core/events.go

```diff
@@ -38,9 +38,6 @@ type PendingLogsEvent struct {
 // PendingStateEvent is posted pre mining and notifies of pending state changes.
 type PendingStateEvent struct{}
 
-// NewBlockEvent is posted when a block has been imported.
-type NewBlockEvent struct{ Block *types.Block }
-
```

Comment:
Because I discovered through this PR that we don't use that event. It's dead code. So I didn't want to open an entire PR just to delete 2 lines of dead code.